elastic/integrations
# [Logstash] Add pipeline level worker utilization

This commit adds support for pipeline level worker utilization from the Logstash API

This involves

* Adding mapping to the fields for the worker_utilization metrics in the pipeline datastream
* Adding exclusions to the ingest pipeline to tidy up the data coming in
* Adding visualizations to the `pipelines` and `single pipeline view` dashboards

Additionally, this PR removes some incorrect flow metrics from the dashboards - previously, the summary tables in the pipeline details view would show flow metrics _summing_ all data across the picker time period, giving a nonsense value.


## Checklist



## How to test this PR locally

Install the integration, and point to a recent version of Logstash. Verify that the `worker_utilization` visualizations are present, and reflect the data from the version of Logstash

## Related issues

Relates #10028  
Closes #10029
 

## Screenshots
Worker Utilization in `Pipelines Overview`
<img width="1206" alt="Screenshot 2024-06-07 at 2 35 59 PM" src="https://github.com/elastic/integrations/assets/4061337/5826e5ac-625b-47c1-9ab9-537fe0c54a19">

Worker Utilization in `Single Pipeline View`
<img width="1201" alt="Screenshot 2024-06-07 at 2 36 13 PM" src="https://github.com/elastic/integrations/assets/4061337/87a64894-d73c-4ace-bc47-e14398647b67">
